annotation/app/models.py

```diff
@@ -244,3 +264,69 @@ class ManualAnnotationTask(Base):
     user = relationship("User", back_populates="tasks")
     jobs = relationship("Job", back_populates="tasks")
     docs = relationship("AnnotatedDoc", back_populates="tasks")
+    stats = relationship("AnnotationStatistics", back_populates="task")
+    agreement_score = relationship(
+        "AgreementScore", uselist=False, back_populates="task"
+    )
+
+
+class AnnotationStatistics(Base):
+    __tablename__ = "annotation_statistics"
+    task_id = Column(
+        INTEGER,
+        ForeignKey("tasks.id", ondelete="cascade"),
+        nullable=False,
+        primary_key=True,
+    )
+    task = relationship("ManualAnnotationTask", back_populates="stats")
+    event_type = Column(
+        ENUM(AnnotationStatisticsEventEnumSchema, name="event_type"),
+        nullable=False,
+        default=AnnotationStatisticsEventEnumSchema.opened,
+    )
+    created = Column(DateTime(), server_default=func.now())
+    updated = Column(DateTime(), server_onupdate=func.now())
+    additional_data = Column(JSONB, nullable=True)
+
+    # TODO validate even_type not "closed" on create instance
+    def to_dict(self) -> dict:
+        return {
+            "task_id": self.task_id,
+            "event_date": self.event_date,
+            "event_type": self.event_type,
+            "additional_data": self.additional_data,
+        }
+
+
+class AgreementScore(Base):
+    __tablename__ = "agreement_score"
+    annotator_id = Column(
+        UUID(as_uuid=True),
+        ForeignKey("users.user_id"),
+        nullable=False,
+        primary_key=True,
```

Comment:
So agreement score is only one per annorator?
And I believe you may omit `nullable=False` since primary key is treated by sqlalchemy as so

Reply:
Yes, thats wrong. One score per task, of course.